Makefile

```
@@ -5,14 +5,14 @@ setup_dev_environment:
 	poetry run pre-commit install
 
 test:
-	poetry run python -m pytest de_workloads/ingest/Ingest_AzureSql_Example/tests/unit
+	poetry run python -m pytest de_workloads/ingest/ingest_azure_sql_example/tests/unit
 	poetry run python -m pytest de_workloads/processing/silver_movies_example/tests/unit
 	poetry run python -m pytest de_workloads/processing/silver_movies_example_with_data_quality/tests/unit
 	poetry run python -m pytest de_workloads/processing/gold_movies_example/tests/unit
 	poetry run python -m pytest datastacks/tests/unit
```

Can we not just simplify this command to have pytest gather tests dynamically?
I've tried it and it successfully gathered all the tests and executed them in about 42s compared to 57s

```suggestion
test:
	poetry run python -m pytest de_workloads
	poetry run python -m pytest datastacks
```

---

Yeah happy with that suggestion